src/js/pages/ClientMirror.js

```diff
@@ -108,4 +156,5 @@ class ClientMirror extends React.Component {
   }
 }
 
-export default ClientMirror;
+
+export default withRouter(ClientMirror);
```

Comment:
these `withRouter` part are needed to do the redirect. 
https://reacttraining.com/react-router/web/api/withRouter